config/locales/ca.yml

```diff
@@ -3,6 +3,7 @@ ca:
   activemodel:
     attributes:
       challenge:
+        area_id: Territori
```

Comment:
This translation tag was removed in favour of 'scope':
https://github.com/gencat/decidim-challenges/blob/302f5c7af9c04114fc2e91b5ed1052d4dd3720ef/config/locales/ca.yml#L11